cli/src/cancel_process_instance.rs

```diff
@@ -0,0 +1,34 @@
+use async_trait::async_trait;
+use color_eyre::eyre::Result;
+
+use clap::Args;
+use zeebe_client::{
+    api::{CancelProcessInstanceRequest, CancelProcessInstanceResponse},
+    ZeebeClient,
+};
+
+use crate::ExecuteZeebeCommand;
+
+#[derive(Args)]
+pub(crate) struct CancelProcessInstanceArgs {
+    process_instance_key: i64,
+}
+
+impl From<&CancelProcessInstanceArgs> for CancelProcessInstanceRequest {
+    fn from(args: &CancelProcessInstanceArgs) -> Self {
+        CancelProcessInstanceRequest {
+            process_instance_key: args.process_instance_key,
+        }
+    }
+}
+
+#[async_trait]
+impl ExecuteZeebeCommand for CancelProcessInstanceArgs {
+    type Output = CancelProcessInstanceResponse;
+    async fn execute(self, client: &mut ZeebeClient) -> Result<Self::Output> {
```

Comment:
:thought_balloon: Do we want to inline the output type? (would also apply to other places)

Reply:
Do you mean `-> Result<CancelProcessInstanceResponse>`?

I'd consider that personal preference and lean towards keeping as as but :shrug:

Reply:
Yes, I think we should keep it as is. the output type is required by `ExecuteZeebeCommand`